doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,248 @@
+| server_templates | List | Yes |  |
+| server_infos | List | Yes |  |
+
+<p align="center"><b>Table 5-3-1:</b> Hardware resource description.</p>
+
+### 5.3.2 Server template
+The server template describes the capability of the host like processor, memory, harddrive, raid, NIC/NIC binding, manufacturer, model etc.
+Server template would be assigned to multiple servers, i.e physical hosts.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| template_name | String | Yes |  |
+| manufacturer | String | Yes |  |
+| model | String | Yes |  |
+| processor | String | Yes |  |
+| memory | String | Yes |  |
+| hard_drive | String | Yes |  |
+| raid | String | Yes |  |
+| network_controller_infos | List | Yes |  |
+| network_card_bond_infos | List | No | NIC bonding might not be always the case.|
+
+<p align="center"><b>Table 5-3-2:</b> Server template.</p>
+
+### 5.3.3 Server information
+Server will reference a server template, i.e, inherting all configuration of server template,
+Besides it may include additional information pim username, password, rack_name,position etc.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| template_name | String | Yes | template referenced |
+| device_name | String | Yes | e.g NFV-D-HDBNJ-02A-3503-G-02-M-SRV-01 |
+| outband_ip | String | Yes |  |
+| outband_subnet_mask | String | Yes |  |
+| pim_username | String | Yes |  |
+| pim_password | String | Yes |  |
+| rack_name | String | Yes |  |
+| position | String | Yes |  |
+| remote_manange_ip | String | Yes | remote management ip, e.g. iLO,iDRAC, BMC |
+| remote_user | String | Yes | remote user  |
+| remote_password | String | Yes | remote password |
+
+<p align="center"><b>Table 5-3-3:</b> Server information.</p>
+
+
+### 5.3.4 Software configuration definition
+It includes Virtualized Infrastructure Manager configurations,
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| vim_name | String | Yes | vim name |
+| vim_id | String | No | a name VIM Resource pool naming rule |
+| vendor | String | Yes | VIM Provider information, e.g. ZTE,Huawei,Ericsson,NOKIA|
+| version | String | Yes | e.g NFV-D-HDBNJ-02A-3503-G-02-M-SRV-01 |
+| ip_version | String | Yes | Ipaddress type: IPV6 or IPV4 |
+| az_info | List | Yes | list of availability zone name designed for this VIM |
```

Comment:
AZ and HA info should belong to compute section.

Reply:
That's correct, I would remove az_info and ha_info from here. Since compute node and control node having such attribute.